src/bioregistry/data/bioregistry.json

```
@@ -38748,6 +38748,24 @@
       "uri_format": "http://www.metanetx.org/equa_info/$1"
     }
   },
+  "metatlas.reaction": {
+    "contact": {
+      "name": "Mihail Anton",
+      "orcid": "0000-0002-7753-9042"
+    },
+    "contributor": {
+      "github": "@mihai-sysbio",
+      "name": "Mihail Anton",
+      "orcid": "0000-0002-7753-9042"
+    },
+    "description": "Metabolic atlas is a resource for exploring metabolism, starting with a set of of community-curated genome-scale metabolic models of human and model organisms, enriched with pathway maps and other tools for easy browsing and analysis.",
+    "example": "MAR11851",
+    "github_request_issue": 495,
+    "homepage": "https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR11851",
+    "name": "Metabolic Atlas",
```

I didn't notice at first, but this should probably be:
```suggestion
    "name": "Metabolic Atlas Reaction",
```